src/nft/components/explore/Carousel.tsx

```diff
@@ -8,18 +8,13 @@ import styled, { css } from 'styled-components/macro'
 const MAX_CARD_WIDTH = 530
 
 const carouselHeightStyle = css`
-  height: 315px;
+  height: 296px;
 
-  @media only screen and (max-width: ${({ theme }) => `${theme.breakpoint.md}px`}) {
-    height: 296px;
+  @media only screen and (min-width: ${({ theme }) => `${theme.breakpoint.md}px`}) {
+    height: 316px;
```

Comment:
Any particular reason for this revert? I think other media queries within the application go from highest screen through `max-width`, so I'd love to keep this the same, unless there's a good reason for doing otherwise!

Reply:
We currently have a mix of both, but mobile defaults with increasing over-rides is the preferred method. (Hasn't been enforced v well.)

Reply:
as stated elsewhere i think we can discuss min vs max advantages elsewhere. really depends if it matters and if we build for mobile first.